### PR TITLE
fix: add measurements

### PR DESCRIPTION
**Pull request by wadjih-bencheikh18**
closes https://github.com/zakodium-oss/analysis-ui-components/issues/342

### CODE REVIEW

File: src/app-data/appState.tsx [hunk 1]
```diff
@@ -100,22 +101,22 @@ function actionHandler(draft: Draft<AppState>, action: AppStateAction) {
       return;
     case 'ADD_MEASUREMENTS': {
       mergeMeasurements(draft.data.measurements, action.payload);
-      for (let kind of Object.keys(action.payload) as MeasurementKind[]) {
-        if (!draft.view.selectedMeasurements[kind]) {
-          const measurement = getFirstMeasurementOrFail(
+
+      for (const kind of Object.keys(kindsLabel).filter(
+        (k) => k in action.payload,
+      ) as MeasurementKind[]) {
```
Comment (targos) [inline src/app-data/appState.tsx:107]: What's the problem with `Object.keys(action.payload)` ?

Comment (wadjih-bencheikh18) [reply src/app-data/appState.tsx:107]: it's not ordered, in Measurement panel we use the kinds order of kindsLabel

Comment (wadjih-bencheikh18) [reply src/app-data/appState.tsx:107]: when we add measurements if no one is selected we have to select the first one in the first kind

Comment (targos) [reply src/app-data/appState.tsx:107]: OK, thank you for the explanation!

**Pull request merged**